docs/paas-plus/dev/platform-as-a-service-plus.md

```diff
@@ -0,0 +1,161 @@
+2. getting started in Spryker CI
+
+
+## Step 1: Getting your project ready for PaaS Upgradability  
+For our current version of the upgradability package, the following pre-requisites apply:
+
+//need better formating for the pre-requisites
+
+**1. Customer should be on the latest major version available for a module** 
+**Why?** Spryker upgrader service will automatically provide minor and patches. If your application is not on the latest major of a module, you will only be able to receive minor and patches that apply to the latest major version you have upgraded to.
+
+**2. Customer should be compliant with Spryker Quality Gate**
+**-- Why?** Spryker Quality Gate contains code checkers to ensure code compliance with Srpyker development and customization best practices. These code checkers ensure that your Spryker application is compatible with minor and patch updated, even when it’s highly customized. 
+**How can I know if my project is compliant?** Please read our documentation on [Keeping a project upgradable] (https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/keeping-a-project-upgradable.html#select-a-development-strategy). You can check if your customized project is compliant with Spryker Quality Gate by using our [tool] (https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/keeping-a-project-upgradable.html#check-if-project-is-upgradable-using-the-evaluator-tool). Follow our [guidelines] (https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/upgradability-guidelines.html) in order to solve any non-compliance issues.
+
+**3. Customer should develop their Spryker application using Spryker SDK**
+**Why?** Customers need to use Spryker SDK throughout the development life cycle, to ensure compliance with Spryker architecture and customization best practices. 
+
+**4. Customer should have full E2E test coverage in place**
+**Why?** Spryker upgrades are provided as a PR that is automatically created in the customer’s repository. At the moment, customers should review the PR and test it before merging the new updates, to ensure all business functionality works as expected.
+
+**5. Codebase is on GitHub or GitLab **
+**Why?** At the moment, we only support upgrades on repositories using GitHub or GitLab. If your project uses a different git provider, please reach out to your CSM to ensure we can incorporate it in our product roadmap.
+
+**6. Customer should be headless (optional)**
+**Why?** Currently, the Spryker quality gate does not check compliance of FE customizations with Spryker best practices. You can either move to headless or apply frontend upgrades manually.
+
+//needs better formating
+Disclaimer: As we evolve the product, we will be working on extending functionalities and pre-requirements will be adjusted accordindly.
+
+### Assumptions and Exceptions
```

Comment:
Since this is obligatory, shall we put it as the first prerequisite?

Reply:
Agree! Let's move it as a pre-requirement